fabric-levelgenerators-v1/src/main/java/net/fabricmc/fabric/api/levelgenerator/v1/CreateLevelGenerator.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.levelgenerator.v1;
+
+import java.util.function.Function;
+
+import net.minecraft.util.Identifier;
+import net.minecraft.world.World;
+import net.minecraft.world.biome.source.BiomeSource;
+import net.minecraft.world.gen.chunk.ChunkGenerator;
+import net.minecraft.world.gen.chunk.ChunkGeneratorType;
+import net.minecraft.world.level.LevelGeneratorType;
+
+import net.fabricmc.fabric.impl.levelgenerator.FabricLevelGeneratorType;
+
+public final class CreateLevelGenerator {
+	/**
+	 * Called to create new instance of the LevelGeneratorType.
+	 *
+	 * @param name                identifier of the {@link LevelGeneratorType}
+	 * @param generatorType       instance of {@link ChunkGeneratorType}
+	 * @param biomeSourceSupplier provide {@link BiomeSource} using lambda expression
+	 */
+	public static LevelGeneratorType create(Identifier name, ChunkGeneratorType<?, ? extends ChunkGenerator<?>> generatorType, Function<World, BiomeSource> biomeSourceSupplier) {
+		return FabricLevelGeneratorType.create(name, name, 0, generatorType, biomeSourceSupplier);
+	}
+
+	/**
+	 * Called to create new instance of the LevelGeneratorType.
+	 *
+	 * @param name                identifier of the {@link LevelGeneratorType}
+	 * @param version             version of {@link LevelGeneratorType}
```

Comment:
It would be helpful to clarify for users of the API what the version field does. Is it used in the level.dat, is it used for some internal tracking, etc